app/controllers/users/reset_passwords_controller.rb

```diff
@@ -43,7 +45,13 @@ def edit
     # PUT /resource/password
```

Comment:
as long as we're here, I think this comment is a lie, I would remove it? The route is `PATCH/POST/PUT /users/password` it looks like
```suggestion
```